README.rst

```diff
@@ -55,7 +55,7 @@ For more help on using Mesa, check out the following resources:
 * `Email list`_
 * `PyPI`_
 
-.. _`Intro to Mesa Tutorial` : http://mesa.readthedocs.org/en/latest/intro-tutorial.html
+.. _`Intro to Mesa Tutorial` : http://mesa.readthedocs.org/en/latest/tutorials/intro_tutorial.html
```

Comment:
Omg... I swear I checked it and it wasn't working for me.
Now it is!
Okay i will merge.

Thank you for your patience!!!

On Sat, Jun 25, 2016 at 4:45 PM, Luke Meyers <notifications@github.com>
wrote:

> In README.rst
> <https://github.com/projectmesa/mesa/pull/267#discussion_r68495593>:
>
> > @@ -55,7 +55,7 @@ For more help on using Mesa, check out the following resources:
> >  * `Email list`_
> >  * `PyPI`_
> >
> > -.. _`Intro to Mesa Tutorial` : http://mesa.readthedocs.org/en/latest/intro-tutorial.html
> > +.. _`Intro to Mesa Tutorial` : http://mesa.readthedocs.org/en/latest/tutorials/intro_tutorial.html
>
> The link works for me. Looks like it's redirecting to
> http://mesa.readthedocs.io/en/latest/tutorials/intro_tutorial.html
> instead, though. Does that one work for you?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/projectmesa/mesa/pull/267/files/e5ae7ddea54367efc1aedb0db1b510e84ac0567c#r68495593>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAKLTimKUga-THFjrSCoJZKprwk6CGuKks5qPb2wgaJpZM4I9RO4>
> .
>



-- 
Jacqueline Kazil | @jackiekazil